src/components/RecipeItem.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image"
+import Link from "next/link"
+import { HeartIcon } from "@/components/icon/HeartIcon"
+import { TrendingRecipeType } from "@/types/types"
+
+type Props = {
+  recipe: TrendingRecipeType
+}
```

Comment:
`FeaturedChef` と同様の理由で
```suggestion
type Props = {
  title: string
  description: string
  favoriteCount: number
  image: string
  path: string
}
```

とできると思います